packages/ui-kit/src/components/BareIconButton/BareIconButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { VFC, ReactNode, ButtonHTMLAttributes } from "react";
+import clsx from "clsx";
+
+import styles from "./BareIconButton.module.css";
+import { Button, ButtonProps, ButtonLabel } from "../Button/Button";
+
+export interface BareIconButtonProps
+  extends ButtonHTMLAttributes<HTMLButtonElement> {
+  icon: ReactNode | string;
+}
+
+export const BareIconButton: VFC<BareIconButtonProps> = ({
```

Comment:
Tbh I think this should just be IconButton with `variant="bare"`, since the styling differences are insignificant to the fact that they are both icon buttons (and ulike chip and chip button they do the same job really)

Reply:
Done.